crates/chia-datalayer/src/lib.rs

```diff
@@ -0,0 +1,42 @@
+use std::collections::HashMap;
+
+type Index = usize;
+type Key = Vec<u8>;
```

Comment:
is the length of the key fixed? e.g. a 32 byte hash.
If so, you can make it `[u8; 32]`, which makes it allocated "in-line". i.e. not heap allocated.

Reply:
Yeah, I think it does end up being that in this case.  it's really the key hash.  Thanks.